test/integration/full/landmark-complementary-is-top-level/landmark-complementary-is-top-level-pass.js

```diff
@@ -0,0 +1,35 @@
+
+describe('landmark-complementary-is-top-level test pass', function () {
+	'use strict';
+	var results;
+	before(function (done) {
+		window.addEventListener('load', function () {
+			axe.run({ runOnly: { type: 'rule', values: ['landmark-complementary-is-top-level'] } }, function (err, r) {
```

Comment:
I bet we could cut down on the number of test files for these rules by making them more dynamic. Could you iterate over a list of relevant landmark rules with JavaScript and run the tests that way? It would be less boilerplate and less code to maintain. There's some precedent in the rule runner, although it's more complicated than you would probably need: https://github.com/dequelabs/axe-core/blob/develop/test/integration/rules/runner.js#L101

In the HTML fixtures, if you needed different markup regions you could potentially just put them into different named DIVs and use include/exclude in the axe.run config.

Reply:
I would rather work on some more rules at this point, it seems that we could come back to this issue  when we have more time